layouts/partials/header.html

```diff
@@ -73,10 +75,14 @@
   </div>
   {{ end }}
 
-  <div class="switcher-mobile">
-      {{ partial "language-switcher1.html" . }}
-  </div>
+  {{ if ne .Type "nvrf" }}
+    <div class="switcher-mobile">
+        {{ partial "language-switcher1.html" . }}
+    </div>
+  {{ end }}
```

Comment:
Let's remove the condition here. We want the language switcher to show up when we do create a translation.